src/simpleinterfaces.cpp

```diff
@@ -63,229 +132,251 @@ using namespace finufft::common;
 
 // Dimension 1111111111111111111111111111111111111111111111111111111111111111
 
-int FINUFFT1D1(BIGINT nj, FLT *xj, CPX *cj, int iflag, FLT eps, BIGINT ms, CPX *fk,
-               finufft_opts *opts)
-//  Type-1 1D complex nonuniform FFT. See ../docs/usage.rst
+int finufft1d1many(int n_transf, i64 nj, f64 *xj, c128 *cj, int iflag, f64 eps, i64 ms,
```

Comment:
potentially we can have a member inside finufft_opts, 'using data_type = f64' and then `guruCall<finufft_opts::data_type`. Standard library practise to reduce errors

Reply:
That only works if we change `finufft_opts` into a template, which we cannot do, since it must be plain C.